parcels/particle.py

```diff
@@ -457,6 +473,94 @@ def show(self, **kwargs):
         plt.show()
         plt.pause(0.0001)
 
+    def show_velocity(self, **kwargs):
+        # time at which to plot velocity
+        t = kwargs.get('t', None)
+        if t is None:
+            t = self.particles[0].time
+        # flag to drawing land on plot
+        land = kwargs.get('land', False)
+        # plot domain latitude longitude parameters
+        latN = kwargs.get('latN', np.nan)
+        latS = kwargs.get('latS', np.nan)
+        lonE = kwargs.get('lonE', np.nan)
+        lonW = kwargs.get('lonW', np.nan)
```

Comment:
I can see that is some cases it would be good for a user to specify the plotting domain, but perhaps cleaner to do that with one keyword? E.g. `domain =[lonW, lonE, latS, latN]`? 

Also, this is again something that would be nice to have in `pset.show` too. So for example you could make quiver plot when a user calls `pset.show(field=vectors)` or something like that.